STACpopulator/log.py

```diff
@@ -98,6 +101,12 @@ def filter(self, record: logging.LogRecord) -> bool | logging.LogRecord:
         return record.levelno <= logging.INFO
 
 
+file_handler = {
+    "class": "logging.FileHandler",
+    "level": "INFO",
+    "formatter": "json",
```

Comment:
Given that no `.jsonl` file name is used by default anymore, I think users will get an unexpected result when their logs are not traditional messages, but per-line JSON. Maybe this should be clarified in the option's description, or with a separate option (see other comment)?